finda-auth/src/main/kotlin/finda/findaauth/adapter/in/grpc/AuthGrpcService.kt

```
@@ -0,0 +1,89 @@
+
+        val tokens = getDeviceTokenService.getAllByUserIds(userIds)
+
+        DeviceTokenListResponse.newBuilder()
+            .addAllTokens(tokens.map(::mapToken))
+            .build()
+    }
+
+    private fun parseUUID(value: String): UUID =
+        try {
+            UUID.fromString(value)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)